config/crds.yml

```diff
@@ -427,6 +427,12 @@ spec:
                                                 fieldPath:
                                                   description: 'Required: Selects a field of the app: only annotations, labels, uid, name and namespace are supported.'
                                                   type: string
+                                                kubernetesVersion:
+                                                  description: 'Optional: Get Kubernetes version, defaults (empty) to retrieving the version from the cluster. Can be manually overridden to a value instead.'
+                                                  properties:
+                                                    version:
```

Comment:
this exactly UX is not what we will end up on, just did it for simplicity / speed

Reply:
i would recommend following api contract:

```
- name: ver
  kubernetesVersion: {}
```
more advanced case where overriding happens
```
- name: ver
  kubernetesVersion:
     version: "1.2.3"
```